Makefile

```diff
@@ -29,6 +42,18 @@ run-test:
 	@cd $(root_path)/code/go/0chain.net && go test -tags bn256 -cover ./...
 	@echo "Tests completed."
 
+install-mockery:
+	@([ -d ./tmp/mockery ] || mkdir -p ./tmp/mockery) \
+	&& echo "[+]download mockery" \
+	&& ([ -f ./tmp/mockery/mockery.tar.gz ] || curl -L -o ./tmp/mockery/mockery.tar.gz https://github.com/vektra/mockery/releases/download/v2.12.2/mockery_2.12.2_$(detected_OS)_$(detected_ARCH).tar.gz) \
+	&& echo "[+]install mockery" \
+	&& tar zxvfC ./tmp/mockery/mockery.tar.gz ./tmp/mockery \
+	&& cp ./tmp/mockery/mockery $(GOPATH)/bin/ \
+	&& rm -rf ./tmp/mockery
```

Comment:
This will create an empty `tmp` dir on the project after installing mockery, please remove the whole `tmp` dir after installation since only mockery uses it.